src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -358,6 +360,23 @@ def read(name: str, namespace: str) -> "RobustaPod":
         """Read pod definition from the API server"""
         return Pod.readNamespacedPod(name, namespace).obj
 
+    @staticmethod
+    def wait_for_pod_ready(pod_name: str, namespace: str, timeout: int = 60) -> "RobustaPod":
+        """
+        Waits for the pod to be in Running state
+        """
+        for _ in range(timeout):  # retry for up to timeout seconds
+            try:
+                pod = RobustaPod().read(pod_name, namespace)
```

Comment:
The () is probably unnecessary here for the static method, but ill merge with it.

Reply:
@RoiGlinik you need it. I tried without and had issues.

I think `read` is an instance method.